main.sql

```diff
@@ -35,8 +35,9 @@ CREATE TABLE mag.affiliations(
     AffiliationId BIGINT PRIMARY KEY,
     Rank INT,
     NormalizedName VARCHAR(150),
-    DisplayName VARCHAR(150),
+    DisplayName TEXT,
```

Comment:
I'm assuming there's no reason these need to be VARCHARS but I only changed the ones that didn't fit.